client/pages/me/me.wxml

```diff
@@ -1,2 +1,54 @@
 <!--pages/me/me.wxml-->
-<text>pages/me/me.wxml</text>
+<view class="container">
+
+
+  <view class="userinfo">   <!-- avatar and username -->
+    <image  class="userinfo-avatar" src="{{userInfo.avatarUrl}}" mode="cover"></image>
+    <text class="userinfo-nickname">{{userInfo.nickName}}</text>
+  </view>
+
+
+  <view class="body">   <!-- info -->
+    <view class="info">   <!-- Chinese name -->
+      <text>Chinese name</text>
+      <view class="infoInput">
+          <input wx:if="{{!personal.data.ChineseName}}" placeholder='Tap here to enter' bindinput='chineseNameInput'/>
+        <!-- waiting for server link -->
+        <input wx:else placeholder-class='phText' placeholder='{{chineseName}}' bindinput='chineseNameInput'/>
```

Comment:
Instead of setting the placeholder, maybe we can set the value directly, so that we don't need two variables for each field (chineseName and chineseNameInputValue). It makes the code much simpler. What do you think?

Reply:
If so we will not be able to make "Tap here to enter" text disappear when users tap to input (as far as I know). I think this can improve user experience but I'm okay with both ways.

Reply:
You mean the placeholder will still be there even after the user inputs the text?

Reply:
If we use value instead of placeholder, users must first delete manually the text "Tap here to enter" as it is seen as the initial input value and is showing in the input box.

Reply:
You can set the value to the string it retrieved from the server, and still set the placeholder to "Tap here to enter". So if the value is empty, the placeholder will be shown.

Reply:
okay!